lib/android/app/src/main/java/com/reactnativenavigation/views/StackLayout.java

```diff
@@ -15,6 +20,8 @@
 public class StackLayout extends RelativeLayout implements TitleBarButton.OnClickListener {
 
     private final TopBar topBar;
+    private FabOptionsPresenter fabOptionsPresenter;
```

Comment:
Should the stack contain `FabOptionsPresenter` ?
I think every node can contain a fab, so `FabOptionsPresenter` can be a property of `ViewController` and `fabOptionsPresenter.present(...)` can be called from `ViewController.applyOptions(Options options)`